Sources/Internal/FrameProvider.swift

```diff
@@ -27,13 +27,15 @@ final class FrameProvider {
     size: CGSize,
     layoutMargins: NSDirectionalEdgeInsets,
     scale: CGFloat,
-    monthHeaderHeight: CGFloat)
+    monthHeaderHeight: CGFloat,
+    monthFooterHeight: CGFloat = 0.0)
```

Comment:
this can be `CGFloat?`. We can probably leave out the default value since we always initialize this with all values anyways.

Reply:
Updated to optional